tests/GoTest.sh

```diff
@@ -47,11 +47,11 @@ cp -a ./go_test.go ./go_gen/src/flatbuffers_test/
 # flag -test.bench and the wildcard regexp ".":
 #   go -test -test.bench=. ...
 GOPATH=${go_path} go test flatbuffers_test \
-                     --test.coverpkg=github.com/google/flatbuffers/go \
+                     --coverpkg=github.com/google/flatbuffers/go \
```

Comment:
This was necessary for me...`test.coverpkg` was reported as not registered. I'm not sure if `go test` itself interprets the flag as part of compilation and doesn't actually pass it along to the test binary or something.

`--bench` and `--benchtime` below were changed for consistency, but I'm also happy to change them back. It works either way for me.